pkg/eval/value_test.go

```diff
@@ -16,6 +16,11 @@ var reprTests = []struct {
 	{"a\nb", `"a\nb"`},
 	{"foo bar", "'foo bar'"},
 	{"a\x00b", `"a\x00b"`},
+	{"c\u0000d", `"c\x00d"`},
+	{"a\033b", `"a\eb"`},
+	{"c\u001bd", `"c\ed"`},
+	{"x\302\222\302\214y", `"x\u0092\u008cy"`},
+	{"x\u0092\u008cy", `"x\u0092\u008cy"`},
```

Comment:
This test is not intended to be an exhaustive test against the quoting mechanism; that is the responsibility of the test in the `parse` package. If you think the `parse` package test could use more test cases, add them there instead.

Reply:
I see your point. My first two tests are not a test of the Elvish quoting mechanism. They are a lower level test that the repr functionality produces the expected output.  You're correct that one of the two tests of the `\e` encoding should be removed since it is redundant. And the UTF-8 sequence, and Unicode codepoint, to Unicode codepoint tests belong in the parse package unit test so I'll move them there.